otherlibs/stdune/bin.mli

```diff
@@ -21,6 +21,3 @@ val exists : Path.t -> bool
 
 (** Look for a program in the PATH *)
 val which : path:Path.t list -> string -> Path.t option
-
-(** "make" program *)
-val make : path:Path.t list -> Path.t option
```

Comment:
Was this just unused?

Reply:
Yes it was. If we had a warning that this code remain unused, we probably would have caught the bug in the PR that introduced it.

Reply:
Yeah, sad that we don't have it.